aws-sqs-queue/variables.tf

```diff
@@ -0,0 +1,46 @@
+variable "service_name" {
+  type        = string
+  description = "The name of the service the queues are for."
+}
+
+variable "message_retention_seconds" {
+  type        = number
+  description = "The number of seconds Amazon SQS retains a message."
+  default     = 1209600
+}
+
+variable "message_retention_seconds_ddl" {
+  type        = number
+  description = "The number of seconds Amazon SQS retains a message in the dead-letter queue."
+  default     = 1209600
+}
+
+variable "redrive_policy_count" {
+  type        = number
+  description = "The number of times a message is delivered to the source queue before being moved to the dead-letter queue."
+  default     = 3
+}
+
+variable "fifo_queue" {
+  type        = bool
+  description = "Whether the queue is FIFO."
+  default     = false
+}
+
+variable "deduplication_scope" {
+  type        = string
+  description = "The scope of the deduplication for the messages in the queue."
+  default     = "queue"
+}
+
+variable "fifo_throughput_limit" {
+  type        = string
+  description = "The throughput limit for the FIFO queue."
+  default     = "perQueue"
+}
+
+variable "kms_master_key_id" {
+  type        = string
+  description = "The ID of an AWS-managed customer master key (CMK) for Amazon SQS or a custom CMK."
+  default     = null
+}
```

Comment:
Tohle bych spíš nezadával jako proměnnou, ale použil tady právě `data` jako referenci na ten defaultní sqs klíč. Třeba jako tady: https://github.com/FigurePOS/figure-register-api/blob/514e15098ac2ef6401a94d38e29873190ca2e723/tf/data.tf#L1-L3 a https://github.com/FigurePOS/figure-register-api/blob/514e15098ac2ef6401a94d38e29873190ca2e723/tf/sqs.tf#L15

Reply:
Joo něco takovýho jsem viděl, ale třeba na email service se to tam vůbec neposílá, bo 

`Careful! When the SQS and SNS is encrypted, bounce notifications are not coming. This can be solved if we want in future.`

a nenapadl mě lepší způsob, maximálně jako proměnnou něco jako `encryption` a mít to jako boolean?

Reply:
Jo tu si pamatuju. Hele to si myslím že je výjimka, takže bych na ni spíš ten modul teď prostě nepoužil a spíš se podíval jestli tam můžeme tu enkrypci zapnout.